Makefile

```diff
@@ -150,21 +150,24 @@ deploy: generate_yaml_test
 # images section
 ############################################################
 # Build the docker image
-build-images: test
-	docker build . -f cmd/manager/Dockerfile -t $(REGISTRY)/$(CONTROLLER_IMG):$(VERSION)
+cmd-image:
 	docker build . -f cmd/clusterctl/Dockerfile -t $(REGISTRY)/$(CLUSTERCTL_IMG):$(VERSION)
+controller-image:
+	docker build . -f cmd/manager/Dockerfile -t $(REGISTRY)/$(CONTROLLER_IMG):$(VERSION)
 
-# Push the docker image
-push-images:
-	@echo "push images to $(REGISTRY)"
-	docker push $(REGISTRY)/$(CONTROLLER_IMG):$(VERSION)
+push-cmd-image:
 	docker push $(REGISTRY)/$(CLUSTERCTL_IMG):$(VERSION)
-
-build-push-images:
-	docker build . -f cmd/manager/Dockerfile -t $(REGISTRY)/$(CONTROLLER_IMG):$(VERSION)
+push-controller-image:
 	docker push $(REGISTRY)/$(CONTROLLER_IMG):$(VERSION)
-	docker build . -f cmd/clusterctl/Dockerfile -t $(REGISTRY)/$(CLUSTERCTL_IMG):$(VERSION)
-	docker push $(REGISTRY)/$(CLUSTERCTL_IMG):$(VERSION)
+
+images: test cmd-image controller-image
```

Comment:
s/cmd-image/clusterctl-image

/cc @morvencao

Reply:
I need to update the prow jobs in https://github.com/kubernetes/test-infra/pull/12479